src/MCPServer/lib/unit.py

```diff
@@ -44,8 +44,8 @@ def reloadFileList(self):
         self.fileList = {}
         # currentPath must be a string to return all filenames as bytestrings,
         # and to safely concatenate with other bytestrings
-        currentPath = str(self.currentPath.replace("%sharedPath%", \
-                                                       archivematicaMCP.config.get('MCPServer', "sharedDirectory"), 1) + "/")
+        currentPath = (self.currentPath.replace("%sharedPath%", \
+                                                       archivematicaMCP.config.get('MCPServer', "sharedDirectory"), 1) + "/").encode('utf-8')
```

Comment:
Sounds good, I'll do that. I used `()` not for the newline though, but because this ends in a `+ "/"`; I'll just switch it to `os.path.join`.